data_registry/process_manager/task/process.py

```diff
@@ -71,7 +71,7 @@ def get_status(self):
             self.job.publication_policy = meta.get("publication_policy") or ""
             self.job.ocid_prefix = meta.get("ocid_prefix") or ""
 
-        self.job.context["process_id_pelican"] = compiled_collection["id"]
+        self.job.context["process_compiled_collection_id"] = compiled_collection["id"]
```

Comment:
@jpmckinney I think we can remove this one entirely now as nobody will use it anymore (was required by Pelican, but neither `flattener` nor `coverage` use it)

Reply:
Hmm, wait, Exporter sends it in the job context.